src/org/exist/storage/serializers/Serializer.java

```diff
@@ -894,14 +896,17 @@ public void toSAX(NodeProxy p) throws SAXException {
 	 * 
 	 * &lt;exist:result hits="sequence length" start="value of start" count="value of count">
 	 * 
-	 * @param seq
-	 * @param start
-	 * @param count
+	 * @param seq The sequence to serialize
+	 * @param start The position in the sequence to start serialization from
+	 * @param count The number of items from the start position to serialize
 	 * @param wrap Indicates whether the output should be wrapped
-         * @param typed Indicates whether the output types should be wrapped
-	 * @throws SAXException
+	 * @param typed Indicates whether the output types should be wrapped
+	 * @param compilationTime The time taken to compile the query which produced the sequence
+	 * @param executionTime The time taken to execute the query which produced the sequence
+	 *
+	 * @throws SAXException If an error occurs during serialization
 	 */
-	public void toSAX(Sequence seq, int start, int count, boolean wrap, boolean typed) throws SAXException {
+	public void toSAX(final Sequence seq, int start, final int count, final boolean wrap, final boolean typed, final long compilationTime, final long executionTime) throws SAXException {
```

Comment:
extending an existing method is often a bad thing..... But there are clearly no other users?